packages/contentful--create-contentful-app/src/index.ts

```diff
@@ -197,8 +197,8 @@ async function initProject(appName: string, options: CLIOptions) {
     )
     .option('-a, --action', 'include a hosted app action in the ts or js template')
     .option(
-      '-d, --delivery-function',
-      'include a hosted delivery function template (EAP only, contact monet@contentful.com if interested)',
+      '-f, --function',
+      'include a function template (EAP only, contact monet@contentful.com if interested)',
```

Comment:
Thus is not making sense to me...

Reply:
Monet is the contact for delivery functions only and we already have HAAs with app event functions and entry validation coming next

Reply:
this PR is just for renaming delivery functions, additional support for accepts will be implemented differently. First we want to finish renaming without breaking anything and then implement rest of the changes.

Reply:
Why does it not make sense? I think there is nothing wrong with using `function` instead of `hosted-delivery-function`. Do you prefer keeping `hosted`?

Reply:
no sorry, i don't understand the contact monet@ when you aren't the primary team owning *all the other function capabilities* is what i mean. there's already hosted app actions, too and many more to come shortly, so this feels wrong to land this monet@contentful.com contact information change

Reply:
```suggestion
      'include a function template',
```

Reply:
oh sorry, you are right. We are still in EAP that's why I didn't change this. Sure, updated!